modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPRequest.testcase

```diff
@@ -203,61 +215,35 @@ definition {
 			EVPRequest.goToCreateRequest();
 		}
 
-		task ("Then the 'phone number' field is disabled and it's not possible fill") {
+		task ("Then the 'phone number' field is disabled and it's not possible fill this") {
```

Comment:
Fix to "Then the 'phone number' field is disabled and it's not possible to fill it"